### PR TITLE
Fix/styles review: resolve style-panel/media/grid-column-panel styles reviews

### PR DESCRIPTION
**Pull request by betterdancing**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Background and solution

1. 调整了tabsGroupConfigurator组件的一些颜色问题；
2. 修改所有大写的PX为小写；
3. 工具栏中调整屏幕大小的组件的弹窗内容样式全部丢失，重写了这部分样式；
4. TinyGrid组件的表格列编辑弹窗样式丢失，重写了这部分；
5. 属性面板和样式面板部分Label色值不对，统一修改；
6. 样式面板原有引用的部分颜色变量失效，重新替换；
### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
	- Introduced multiple new UI components including `TinyCarousel`, `TinyDialogBox`, `TinyCollapse`, and `TinyTree`.
	- Enhanced existing components with new properties and event handling capabilities.
	- Added drag-and-drop functionality for background images.
  
- **Bug Fixes**
	- Improved error handling and visibility of error messages in various components.

- **Styling Updates**
	- Updated background colors and text colors for various components to enhance visual consistency.
	- Adjusted layout and spacing for better user experience across multiple components.

- **Documentation**
	- Added localized labels for improved accessibility in Chinese.

- **Chores**
	- Introduced new CSS variables for enhanced theming and styling flexibility.

### CODE REVIEW

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 1]
```diff
@@ -238,8 +237,18 @@ const change = (item) => {
       justify-content: center;
       align-items: center;
       height: 24px;
-      background-color: var(--ti-lowcode-base-bg-5);
+      background-color: var(--te-common-bg-container);
       color: var(--te-common-text-weaken);
+
+      &:hover {
+        background-color: var(--ti-lowcode-base-gray-101);
```
Comment (wenmine) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue]: 这里，不要用-ti-lowcode-base-里的变量了，用common的。

File: packages/toolbars/media/src/Main.vue [hunk 2]
```diff
@@ -196,8 +187,8 @@ export default {
           maxWidth: '1920px',
           title: '大屏',
           subTitle: '1200px 及以上',
-          content: '此处添加的样式将适用于 1200 像素及以上，除非它们在更大的断点处进行编辑',
-          enContent: 'Style added here will apply at 1200px and up,unless they′re edited at a larger breakpoint'
+          content: '此处添加的样式将适用于 1200 像素及以上，除非它们在更大的断点处进行编辑。',
+          enContent: 'Style added here will apply at 1200px and up,unless they′re edited at a larger breakpoint.'
```
Comment (chilingling) [inline packages/toolbars/media/src/Main.vue:186]: 国际化为什么要这样配置？约定不都是 

``
{
   zh_CN: '',
    en_US: ''
}
``

的结构嘛？

File: designer-demo/public/mock/bundle.json [hunk 3]
```diff
@@ -9823,8 +9823,7 @@
                             },
                             "description": {
                               "zh_CN": "单元格编辑渲染配置项，也可以是函数 Function(h, params)"
-                            },
-                            "labelPosition": "left"
```
Comment (chilingling) [inline designer-demo/public/mock/bundle.json:9827]: `packages/engine-cli/template`  下面的 bundle.json 也需要同步改

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 4]
```diff
@@ -19,6 +19,7 @@
             <tiny-popover
```
Comment (chilingling) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue]: 这个 `popover` 的效果只是 hover 提示 content 是吧？用原生 `title` 属性替换行不行？

File: packages/settings/styles/src/components/background/BackgroundImageGradient.vue [hunk 5]
```diff
@@ -12,9 +12,9 @@
     </div>
     <div class="repeat">
       <div>
-        <tiny-checkbox v-model="state.repeat" @change="changeRepeat">Repeat</tiny-checkbox>
+        <tiny-checkbox v-model="state.repeat" @change="changeRepeat">调整列宽</tiny-checkbox>
```
Comment (chilingling) [inline packages/settings/styles/src/components/background/BackgroundImageGradient.vue]: 这里为什么要改成 `调整列宽`？

File: packages/toolbars/media/src/Main.vue [hunk 6]
```diff
@@ -32,20 +35,15 @@
...
+        <tiny-popover v-if="isCanvas" width="200" trigger="click" popper-class="toolbar-media-popper">
           <template #reference>
-            <tiny-popover
-              trigger="hover"
-              :open-delay="1000"
-              popper-class="toolbar-right-popover"
-              append-to-body
-              content="画布设置"
-            >
+            <tiny-popover trigger="hover" :open-delay="1000" popper-class="toolbar-right-popover" append-to-body>
```
Comment (chilingling) [inline packages/toolbars/media/src/Main.vue]: 类名建议语义化替换一下，`toolbar-right-popover` 不太对，这里是 media 插件的

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 7]
```diff
@@ -19,6 +19,7 @@
...
               v-if="item?.content"
               :placement="placement"
+              :open-delay="1000"
```
Comment (hexqi) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue]: popover delay 1s太长了，这里是什么场景，一般200/300ms就够了

File: packages/settings/styles/src/components/background/BackgroundImageGradient.vue [hunk 8]
```diff
@@ -12,9 +12,9 @@
...
       </div>
-      <span><svg-icon name="clock-small"></svg-icon></span>
+      <span><svg-icon name="重启"></svg-icon></span>
```
[the remark this answers is not in the document]
Comment (hexqi) [reply packages/settings/styles/src/components/background/BackgroundImageGradient.vue]: 对应icon的文件名改成英文

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 9]
```diff
@@ -2,34 +2,21 @@
...
-            <tiny-popover
-              v-if="item?.content"
-              :placement="placement"
-              :visible-arrow="false"
-              :content="item.content"
-              trigger="hover"
-            >
-              <template #reference>
+            <span class="label-text">
+              <span :title="item?.content">
```
Comment (chilingling) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue]: 为什么要单独包裹一层来显示 `title` ?，能不能直接在上层 `class="label-text"` 这一层放上 `title` 属性？
下同

File: packages/toolbars/media/src/Main.vue [hunk 10]
```diff
@@ -74,18 +67,11 @@
                   </template>
                 </tiny-input>
               </div>
+              <div>
+                <label for="">{{ '自由布局' }}</label>
```
Comment (chilingling) [inline packages/toolbars/media/src/Main.vue]: 这里 `for=""` 属性有什么用？

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 11]
```diff
@@ -238,38 +206,41 @@ const change = (item) => {
...
         border-radius: 4px;
         color: var(--te-common-text-primary);
       }
+      &.selected {
+        background-color: var(--te-common-bg-prompt);
+        color: var(--te-common-text-primary);
+        border-radius: 4px;
+      }
+    }
+    :deep(.tiny-dropdown__title) {
```
Comment (chilingling) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue]: 这里加个 `popper-class="xxx"` ，显式指定一下类，覆盖指定类下的样式

File: packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue [hunk 12]
```diff
@@ -238,38 +206,41 @@ const change = (item) => {
...
+      margin: 0;
+      line-height: 12px;
+      .selected-option {
+        text-align: center;
+      }
+    }
+    :deep(.tiny-dropdown__suffix-inner) {
```
Comment (chilingling) [inline packages/configurator/src/tabs-group-configurator/TabsGroupConfigurator.vue]: 这里直接给 `tiny-dropdown` 加个 `class`属性，然后在该属性下面使用一个 `deep` 就够了。

```less

 :deep(.xxxx) {
  .tiny-dropdown__suffix {
}
  .tiny-dropdown ... {
  }
}
```

**Review by chilingling — DISMISSED** [review]

**Review by rhlin — DISMISSED** [review]

**Review by chilingling — APPROVED** [review]

**Pull request merged**